modules/phase_field/src/materials/DerivativeParsedMaterialHelper.C

```diff
@@ -21,6 +23,8 @@ DerivativeParsedMaterialHelper::DerivativeParsedMaterialHelper(const InputParame
                                                                VariableNameMappingMode map_mode) :
     ParsedMaterialHelper(parameters, map_mode),
     //_derivative_order(getParam<unsigned int>("derivative_order"))
```

Comment:
I implemented all suggestions except this one. I dislike keeping commented code as well, but I'm planning to have a follow up PR very soon to remove ```third_derivatives``` entirely. I have to double check if we removed its use everywhere that is under our control.